src/app/profile/email-verification/email-verification.component.less

```diff
@@ -0,0 +1,26 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.f8-email-verify {
+  .blank-slate-pf {
```

Comment:
Can you create this as a generic fabric8-ui class in the `card.less` file, so that it can be reused? I'd like to start moving away from overwriting PatternFly styles where possible.

Reply:
I am not sure about this class as this class may contain different background in other scenarios

Reply:
blank-slate-pf class is being overwritten in overrides.less. Styles are still overwritten even after making it a generic class in cards.less